### PR TITLE
Build upload refactor

### PR DESCRIPTION
**Pull request by YLivay**

### CODE REVIEW

File: src/app/components/forms/game/build/build.ts [hunk 1]
```diff
@@ -107,6 +107,40 @@ export default class FormGameBuild extends BaseForm<GameBuildFormModel>
...
+	}
+
+	get archiveError() {
+		if (!this.model) {
+			return '';
+		}
+
+		if (this.model.hasError(GameBuild.ERROR_INVALID_ARCHIVE)) {
+			return this.$gettext(
+				`The archive you uploaded looks corrupt, we can't extract it on our end.`
```
Comment (nilllzz) [inline src/app/components/forms/game/build/build.ts]: Would be weird if it was corrupt. I think you meant "corrupted".

File: src/app/components/forms/game/build/build.ts [hunk 2]
```diff
@@ -107,6 +107,40 @@ export default class FormGameBuild extends BaseForm<GameBuildFormModel>
...
+			);
+		}
+
+		if (this.model.hasError(GameBuild.ERROR_PASSWORD_ARCHIVE)) {
+			return this.$gettext(`The archive you uploaded is password protected.`);
```
Comment (nilllzz) [inline src/app/components/forms/game/build/build.ts]: "password-protected", with a hyphen.

File: src/app/components/forms/game/build/build.vue [hunk 3]
```diff
@@ -72,26 +72,26 @@
...
-			<div class="text-center small" v-translate>
-				<strong>Processing build.</strong>
-				It will become available in this release as soon as we're done.
-			</div>
-		</app-expand>
+				<br />
+				<app-progress-bar thin indeterminate active :percent="100" />
+
+				<div class="text-center small" v-translate>
+					<strong>Processing build.</strong>
```
Comment (nilllzz) [inline src/app/components/forms/game/build/build.vue]: We should translate this.

File: src/app/components/forms/game/build/build.vue [hunk 4]
```diff
@@ -117,288 +117,307 @@
...
+						<p v-if="isBrowserBased">
+							<app-jolticon icon="notice" />
+							<strong>
+								<translate>
+									Be sure to remove or block ads from showing in your browser games.
+								</translate>
+							</strong>
 							<translate>
-								Be sure to remove or block ads from showing in your browser games.
+								We serve our own ads and share the revenue with developers. Domain names to block:
```
Comment (nilllzz) [inline src/app/components/forms/game/build/build.vue]: Is "Domain names to block" correct here?

File: src/app/components/forms/game/build/build.ts (hunk 1 above)
Comment (YLivay) [reply src/app/components/forms/game/build/build.ts]: your face looks corrupted

File: src/app/components/forms/game/build/build.vue (hunk 3 above)
Comment (YLivay) [reply src/app/components/forms/game/build/build.vue]: its in a v-translate.

File: src/app/components/forms/game/build/build.vue (hunk 4 above)
Comment (YLivay) [reply src/app/components/forms/game/build/build.vue]: I think so? besides this doesnt have anything to do with this PR. Probably a diff from a merge.

**Pull request merged**